packages/server/api/src/app/flows/flow/flow.service.ts

```diff
@@ -409,7 +402,7 @@ export const flowService = {
           flowId: id,
         });
         throw new ApplicationError({
-          code: ErrorCode.FLOW_OPERATION_INVALID,
+          code: ErrorCode.FLOW_INTERNAL_FORBIDDEN,
```

Comment:
this still needs to be in here, because it is a hard deletion. at some point we will change this to be a soft delete but until then im blocking it here.